### PR TITLE
webuitests: adapt to Random Serial Numbers

### PR DESCRIPTION
**Pull request by flo-renaud**
The webui tests were written for sequential serial numbers and expect the certs to be issued with low serial numbers. Adapt to Random Serial Numbers.

Fixes: https://pagure.io/freeipa/issue/9707

### CODE REVIEW

**Comment by flo-renaud** [comment]
Green run of `test_webui_cert`: [Details](http://freeipa-org-pr-ci.s3-website.eu-central-1.amazonaws.com/jobs/fbc6510e-ac04-11ef-a01a-fa163e59c9b4/)

File: ipatests/test_webui/test_cert.py [hunk 1]
```diff
@@ -226,8 +251,16 @@ def test_search_maximum_serial(self):
         """
         self.init_app()
         self.navigate_to_entity(ENTITY)
+
+        # Find the second lowest serial number
+        result = self.execute_api_from_ui('cert_find', [], {})
+        certs = result['result']['result']
+        serials = [int(cert["serial_number_hex"], 0) for cert in certs]
```
Comment (rcritten) [inline ipatests/test_webui/test_cert.py:258]: There should be a "serial_number" value included in the output which is decimal. It's in the API output anyway.

Comment (flo-renaud) [reply ipatests/test_webui/test_cert.py:258]: I tried to use that in my first attempt but the call sometimes returns the value with a scientific notation (like 3.12345678e+37) which is not precise enough to allow extraction of the real value.

Comment (rcritten) [reply ipatests/test_webui/test_cert.py:258]: Ok, that is probably worth a bug on its own. Hex is fine.

**Comment by flo-renaud** [comment]
master:

* c8befc9f46b43aec748ede33236ca4f77b2356c6 webuitests: adapt to Random Serial Numbers

**Pull request closed without merging**